core/templates/dev/head/pages/exploration_player/AnswerFeedbackPairDirective.js

```diff
@@ -66,6 +66,10 @@ oppia.directive('answerFeedbackPair', [
             }
             return shortAnswerHtml;
           };
+
+          $scope.getRequestForHint = function() {
+            return $scope.data.learnerAnswer;
```

Comment:
Can I use the answer/feedback directive to display the request/hint pair or should that be separate as well ie. should there be a separate request/hint directive?